Mathlib/LinearAlgebra/Matrix/Spectrum.lean

```diff
@@ -55,6 +56,33 @@ lemma mulVec_eigenvectorBasis (j : n) :
       congr(⇑$((isHermitian_iff_isSymmetric.1 hA).apply_eigenvectorBasis
         finrank_euclideanSpace ((Fintype.equivOfCardEq (Fintype.card_fin _)).symm j)))
 
+/--Eigenvalues of a Hermitian Matrix, coerced, belong to the spectrum of the assoc.toEuclideanLin -/
+theorem eigenvalue_mem_toEuclideanLin_spectrum_RCLike (i : n) :
+    (RCLike.ofReal ∘ hA.eigenvalues) i ∈ spectrum 𝕜 (toEuclideanLin A) :=
+  LinearMap.IsSymmetric.hasEigenvalue_eigenvalues _ _ _ |>.mem_spectrum
+
+/-- Algebra equivalence between the linear maps and continuous linear maps on a finite-dim module.-/
+def AlgEquivFiniteDimNormedLinearCLM.{v} (E : Type v) [NormedAddCommGroup E]
```

Comment:
Perhaps this should be called `Module.End.toContinuousLinearMap` to match?

Reply:
Thanks for all the helpful suggestions! Should this result be moved into Algebra.Module._ , or do you think it still fits here?

Reply:
This result should probably move next to `LinearMap.toContinuousLinearMap` if possible. If not, `#find_home` might have some useful suggestions as to where to put it. Certainly it should not need `RCLike`, as `LinearMap.toContinuousLinearMap` doesn't.